### PR TITLE
Suggestion to add first specs

### PR DESCRIPTION
**Pull request by shioimm**
This is a suggestion to add first specs for this gem,
because TraceLocation has no specs yet.
Although there is still no verification for trace method options
(e.g. ignore, format). It might be added by other contexts.
Would you tell me how do you think about this?
Or if you have some better ways, please let me know.

### CODE REVIEW

File: spec/trace_location_spec.rb [hunk 1]
```diff
@@ -1,7 +1,84 @@
 # frozen_string_literal: true
 
+require 'fileutils'
```
Comment (yhirano55) [inline spec/trace_location_spec.rb]: I thought it seems this statement is better to move to spec_helper.rb

File: spec/trace_location_spec.rb [hunk 2]
```diff
@@ -1,7 +1,84 @@
...
+require_relative 'support/dummy/dummy'
```
Comment (yhirano55) [inline spec/trace_location_spec.rb]: I felt it's not necessary to separate files.

Comment (yhirano55) [reply spec/trace_location_spec.rb]: Ah....I understood its concept. Sorry.

**Comment by udzura** [comment]
🎉

File: spec/trace_location_spec.rb (hunk 1 above)
Comment (shioimm) [reply spec/trace_location_spec.rb]: Thank you for this suggestion!(And sorry for the delay in replaying)
I'll move this statement to spec_helper in next PR.

**Pull request merged**